docs/user-guide/install-zowe-smpe.md

```diff
@@ -124,7 +124,14 @@ All issues of previous releases of Zowe that were resolved before August 2019 ha
 
 ### Service level information
 
-Since this is the first release of the SMP/E package, no PTFs have been created.
+The Zowe SMP/E package is a distribution of Zowe version 1.9.0 with an FMID of AZWE001.
+
+Subsequent releases of the Zowe z/OS components are delivered as rollup PTFs and delivered on [zowe.org](https://zowe.org#download).  Because of the file size of the PTF  it is packaged as two co-requisites PTFs which are made available in a single .zip download.  
```

Comment:
Should say `co-requisite` i.e. singular.

Reply:
Fixed this while editing the description.